src/SearchBar/SearchBar.js

```diff
@@ -0,0 +1,56 @@
+import React, { Component } from 'react'
+import './SearchBar.css'
+
+export default class SearchBar extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      searchTerm: '',
+      feedback: ''
+    }
+  }
+
+  sendSearchTerm = (event) => {
+    event.preventDefault()
+    if (this.state.searchTerm) {
+      try {
+        this.props.findMovie(this.state.searchTerm)
+      } catch ({  message }) { // this is the message that's part of error handling
+        this.setState({ searchTerm: '' })
+      }
+    } else {
+      this.setState({ feedback: 'Something went wrong. Please try a new search.' })
+    }
+
+  }
+
+  render() {
+
+    return (
+      <div className="searchbar-container">
+        <h1>Welcome.</h1>
+        <h3>We have all the movies. Explore now.</h3>
+        <form>
+          <label>
+            <input
+              type="text"
+              placeholder="Search for a movie......."
+              name="searchTerm"
+              onChange={(event) => {
+                return this.setState({searchTerm: event.target.value})
```

Comment:
Instead of doing the setState here, you'll probably want to create a change handler. Similar to a controlled form, you'll want to create an attribute "value" to this field, with a value of {this.state.searchTerm}. Then, the onChange will point to a changeHandler function, which will update the SearchBar's state. That function can then call this.props.findMovie.

The onChange would ultimately look like 
`onChange={this.changeHandler}`